### ISSUE TITLE
Are weights of TargetNet module updated directly?

### ISSUE THREAD

**Issue by ifujisawa** [likesiwell/LGM-Net#7]
Hello, I am trying to understand your code especially how to generate weights theta from meta module.

According to the Algorithm 1 in the paper, weights of TargetNet are generated from MetaNet and they are not updated directly by the gradient of loss. When I run train_meta_matching_network.py, however, the thetas (they correspond to the weights of Classifier, that is, last 4 lines below) might be trainable parameters like below.
If it's true, the thetas are not generated in batch loop during training, right?

##############################
The variables of  All trainable variables
extractor/conv_layers/g_conv1/conv2d/kernel:0 (3, 3, 3, 64)
extractor/conv_layers/g_conv1/conv2d/bias:0 (64,)
extractor/conv_layers/g_conv1/BatchNorm/beta:0 (64,)
extractor/conv_layers/g_conv1/BatchNorm/gamma:0 (64,)
extractor/conv_layers/g_conv2/conv2d/kernel:0 (3, 3, 64, 64)
extractor/conv_layers/g_conv2/conv2d/bias:0 (64,)
...
Classifier/meta_conv1/MetaNetwork_meta_conv/meta_weights/w1:0 (32, 36928)
Classifier/meta_conv1/MetaNetwork_meta_conv/meta_weights/b1:0 (36928,)
Classifier/meta_conv2/MetaNetwork_meta_conv/meta_weights/w1:0 (32, 36928)
Classifier/meta_conv2/MetaNetwork_meta_conv/meta_weights/b1:0 (36928,)
##############################

**Comment by ifujisawa** [comment]
Excuse me, finally, I understood I was wrong. All meta_weights are phi in the paper's notation. So there is no problem. Thanks.
